### PR TITLE
NXDRIVE-745: Update documentation accoriding to CI changes

### PR DESCRIPTION
**Pull request by BoboTiG**

### CODE REVIEW

File: DEVELOPERS.md [hunk 1]
```diff
@@ -8,19 +8,17 @@ The projects comes into two parts: the addon deployed on the Nuxeo server, writt
...
+## Building the server addon
 
 To build the nuxeo-drive addon see the related [nuxeo-drive-server](https://github.com/nuxeo/nuxeo-drive-server) GitHub repository.
 
 To build the Marketplace package see the related [marketplace-drive](https://github.com/nuxeo/marketplace-drive) GitHub repository.
 
-## Client-Side Architecture
+## Building the Nuxeo Drive client
+
+See [docs/deployment.md](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/deployment.md).
```
Comment (ataillefer) [inline DEVELOPERS.md]: I think you can use relative links

File: README.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
-# Nuxeo Drive - Desktop Synchronization Client for Nuxeo
+# Nuxeo Drive - desktop synchronization client for Nuxeo
```
Comment (ataillefer) [inline README.md]: Titles should always be capitalized :)

Comment (BoboTiG) [inline README.md]: Done :)

File: DEVELOPERS.md (hunk 1 above)
Comment (BoboTiG) [inline DEVELOPERS.md]: Done.

**Pull request merged**